Exception in shutdown hook of Activemq on service shutdown

An exception is thrown in the logs when the instance is shutdown by an OS restart or shutdown trigger.

```
Exception in thread "ActiveMQ ShutdownHook" java.lang.NoClassDefFoundError: org/apache/activemq/selector/SelectorParser
	at org.apache.activemq.broker.BrokerService.stop(BrokerService.java:769)
	at org.apache.activemq.xbean.XBeanBrokerService.stop(XBeanBrokerService.java:91)
	at org.apache.activemq.broker.BrokerService.containerShutdown(BrokerService.java:2303)
	at org.apache.activemq.broker.BrokerService$6.run(BrokerService.java:2270)
Caused by: java.lang.ClassNotFoundException: org.apache.activemq.selector.SelectorParser
	at java.net.URLClassLoader.findClass(URLClassLoader.java:382)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:419)
	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:352)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:352)
	at org.eclipse.jetty.webapp.WebAppClassLoader.loadClass(WebAppClassLoader.java:538)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:352)
	... 4 more
```

--- a/system/Tools/jetty/defaults/etc/activemq/activemq.xml
+++ b/system/Tools/jetty/defaults/etc/activemq/activemq.xml
@@ -8,7 +8,7 @@ http://www.springframework.org/schema/beans/spring-beans-4.2.xsd
  <!--
         The <broker> element is used to configure the ActiveMQ broker.
     -->
-    <broker useJmx="false" persistent="true" deleteAllMessagesOnStartup="true" xmlns="http://activemq.apache.org/schema/core" brokerName="localhost" dataDirectory="data/activemq-data">
+    <broker useShutdownHook="false" useJmx="false" persistent="true" deleteAllMessagesOnStartup="true" xmlns="http://activemq.apache.org/schema/core" brokerName="localhost" dataDirectory="data/activemq-data">
         <destinationPolicy>
             <policyMap>
               <policyEntries>

